### PR TITLE
DOC-837 Allow trial licenses on RPCN

### PR DESCRIPTION
**Pull request by asimms41**
## Description

Resolves [DOC-837](https://redpandadata.atlassian.net/browse/DOC-837)
Review deadline: December 9th

## Page previews

https://deploy-preview-900--redpanda-docs-preview.netlify.app/current/get-started/licensing/overview/

## Checks

- [ ] New feature
- [x] Content gap
- [ ] Support Follow-up
- [ ] Small fix (typos, links, copyedits, etc)

[DOC-837]: https://redpandadata.atlassian.net/browse/DOC-837?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: modules/get-started/pages/licensing/overview.adoc [hunk 1]
```diff
@@ -36,16 +36,13 @@ Starting with version 24.3, new Redpanda clusters automatically receive a trial
...
-====
-Redpanda Connect does not accept trial license keys. To use enterprise features of Redpanda Connect, https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^] and then xref:redpanda-connect:get-started:licensing.adoc[apply your license key].
-====
-
 [NOTE]
 ====
 include::get-started:partial$licensing/block-upgrades-note.adoc[]
 ====
 
+In Redpanda Connect, you must https://redpanda.com/try-enterprise[generate] and xref:redpanda-connect:get-started:licensing.adoc#apply-a-license-key-to-redpanda-connect[apply a trial license key] to evaluate enterprise features. The 30-day evaluation period begins when you run your first data pipeline. After this period expires, you are blocked from using enterprise connectors unless you upgrade to an https://www.redpanda.com/upgrade[Enterprise Edition license^].
```
Comment (Jeffail) [inline modules/get-started/pages/licensing/overview.adoc]: We aren't able to track when "your first data pipeline" runs, this is why we initially didn't support trial licenses, so instead we're just allowing it to work until the trial license itself expires.

Comment (asimms41) [reply modules/get-started/pages/licensing/overview.adoc]: Updated. Can you also check the information in this section for accuracy?

=== Redpanda Connect enterprise features

**Review by Jeffail — APPROVED** [review]

File: modules/get-started/pages/licensing/overview.adoc [hunk 2]
```diff
@@ -36,16 +36,13 @@ Starting with version 24.3, new Redpanda clusters automatically receive a trial
...
-====
-Redpanda Connect does not accept trial license keys. To use enterprise features of Redpanda Connect, https://redpanda.com/upgrade[upgrade to Redpanda Enterprise^] and then xref:redpanda-connect:get-started:licensing.adoc[apply your license key].
-====
-
 [NOTE]
 ====
 include::get-started:partial$licensing/block-upgrades-note.adoc[]
 ====
 
+In Redpanda Connect, you must https://redpanda.com/try-enterprise[generate] and xref:redpanda-connect:get-started:licensing.adoc#apply-a-license-key-to-redpanda-connect[apply a trial license key] to evaluate enterprise features. After the 30-day evaluation period, you are blocked from using enterprise connectors unless you upgrade to an https://www.redpanda.com/upgrade[Enterprise Edition license^].
```
Comment (JakeSCahill) [inline modules/get-started/pages/licensing/overview.adoc]:
```suggestion
To evaluate enterprise features in Redpanda Connect, you must xref:redpanda-connect:get-started:licensing.adoc#apply-a-license-key-to-redpanda-connect[apply a trial license key]. After the 30-day evaluation period, you are blocked from using enterprise connectors unless you https://www.redpanda.com/upgrade[upgrade to an Enterprise Edition license^].
```

**Review by JakeSCahill — APPROVED** [review]
Left a suggestion but no blockers.

**Review by JakeSCahill — APPROVED** [review]

**Pull request merged**